core/db/migrate/20210902110754_rename_column_access_hash_to_token.rb

```diff
@@ -0,0 +1,8 @@
+class RenameColumnAccessHashToToken < ActiveRecord::Migration[5.2]
```

Comment:
@MatthewKennedy could you rename the migration so they will run after https://github.com/spree/spree/blob/main/core/db/migrate/20210920090344_add_deleted_at_to_spree_stores.rb ?

Reply:
Hi @damianlegawiec I think they are already in that order:
<img width="513" alt="Screenshot 2021-09-21 at 08 04 01" src="https://user-images.githubusercontent.com/1240766/134126344-78d73be0-c760-4ba6-af82-d36f0b726427.png">

Reply:
Scrap that last comment, I've just merged main into this PR and noticed the change... one moment on this fix...

Reply:
Current Order of Migrations
<img width="568" alt="Screenshot 2021-09-21 at 08 12 56" src="https://user-images.githubusercontent.com/1240766/134127420-a6d46450-c370-43b7-9f62-2ff68104d51c.png">